.config/commonjs.js

```diff
@@ -0,0 +1,15 @@
+import { baseConfig } from './base';
+import commonjs from 'rollup-plugin-commonjs';
+
+const env = process.env.NODE_ENV;
+const envHotType = process.env.HOT_TYPE;
+const filename = 'react-handsontable.js';
+
+export const cjsConfig = {
+  output: {
+    format: env,
+    indent: false,
+    file: './commonjs/' + envHotType + '/' + filename
+  },
+  plugins: baseConfig.plugins.concat([commonjs()])
+};
```

Comment:
Could you add an empty line below?

Reply:
It has an empty last line, github just doesn't show it.